scripts/fhir_scripts.sh

```diff
@@ -204,6 +204,108 @@ function igtools_version() {
     fi
 }
 
+function check_pytool_version() {
+    local package_name="$1"
+    local repo_url="$2"
+    local current_version=""
+    local latest_version=""
+    local response=""
+
+    # Handle missing parameters gracefully
+    if [[ -z "$package_name" || -z "$repo_url" ]]; then
+        echo "error:missing_parameters"
+        return 0
+    fi
+
+    # Get current version from pipx
+    current_version=$(sudo pipx list --global --short 2>/dev/null | awk -v pkg="$package_name" '$1 == pkg {print $2}') || true
```

Comment:
What's the reason to parse the pipx list instead of calling the tool with the version parameter?

Reply:
That would mean replacing the already existing logic with this approach. 
`[tool] --version` can return a string formatted in different ways.
I chose sudo pipx list --global --short here 'cause of the unified format.

Yes, principally, those method should be aligned. But is it worth the effort when the scripts get's deprecated soon?